lib/vrf.c

```diff
@@ -1214,7 +1214,8 @@ const struct frr_yang_module_info frr_vrf_info = {
 				.get_next = lib_vrf_get_next,
 				.get_keys = lib_vrf_get_keys,
 				.lookup_entry = lib_vrf_lookup_entry,
-			}
+			},
+			.priority = NB_DFLT_PRIORITY - 2,
```

Comment:
Currently `ospf6-topo1-vrf` (a new topotest) fails b/c ospf tries to look the vrf up. By setting this priority the vrf will get created first, otherwise the lookup can fail. It's probably better to fix the ospfv3 code, and I think you mentioned you would do that? If so we can probably remove this.